spec/ADAS/ADAS.vspec

```diff
@@ -113,6 +124,9 @@ LaneDepartureDetection.IsError:
 #
 ObstacleDetection:
   type: branch
+  instances:
+    - ["Front", "Rear"]
```

Comment:
Do we need the instances also for the existing signals like `IsEnabled`, `IsWarning` and so on? If not we could split to two branches, one called `ObstacleDetection` (without instances) and one called `Obstacles` (with instances)

Reply:
Each ObstacleDetection sensor could throw a separate warning, be disabled independently etc,

Reply:
Previous conversation before considering instances approach

https://github.com/COVESA/vehicle_signal_specification/pull/770